app/views/projects/wikis/history.html.haml

```diff
@@ -12,11 +12,12 @@
       %th Last updated
       %th Format
   %tbody
-    - @page.versions.each do |version|
+    - @page.versions.each_with_index do |version, index|
       - commit = version
       %tr
         %td
-          = link_to project_wiki_path(@project, @page, version_id: commit.id) do
+          = link_to project_wiki_path(@project, @page,
+              index == 0 ? {} : {version_id: commit.id}) do
```

Comment:
can you refactor this? Maybe move `index == 0 ? {} : {version_id: commit.id}` in separate variable